Jenkinsfile.cloud

```diff
@@ -158,7 +158,6 @@ node(NODE) {
 
     stage("Finalizing, generate metadata") {
         // Everything above in parallel worked on qcow, we're done with it now
-        sh "rm ${qcow}"
```

Comment:
Hmm, we should probably gzip this one too. Maybe something like:

```
gzip < ${qcow} > ${qcow}.local.gz
```

?

(Or alternatively rename the openstack one to `${qcow}.openstack.gz` and this one just `${qcow}.gz`).

Reply:
Hmm, how about `${qcow}.openstack.gz` & `${qcow}.qemu.gz`

Reply:
Sure, that works. The `qemu` is a bit misleading since it's not actually explicitly stamped with `coreos.oem.id=qemu`. Though I suppose that's the target use case.